schemas/2019-01-21/Microsoft.Kusto.json

```diff
@@ -230,13 +244,16 @@
       "description": "Class representing the Kusto cluster properties."
     },
     "DatabaseProperties": {
+      "type": "object",
       "properties": {
         "softDeletePeriod": {
           "type": "string",
+          "format": "duration",

```

Comment:
>"format": "duration", [](start = 10, length = 21)

Visual Studio doesn't support the "duration" format. can we just  remove this lines?

Reply:
Removed